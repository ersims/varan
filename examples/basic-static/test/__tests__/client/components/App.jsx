import React from 'react';
import renderer from 'react-test-renderer';
import { App } from '../../../../src/components/App';

// Tests
test('renders correctly', () => {
  const component = renderer.create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
