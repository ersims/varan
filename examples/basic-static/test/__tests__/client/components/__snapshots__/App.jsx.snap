// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <header
    className="header"
  >
    <h1
      className="title"
    >
      Welcome to React that works awesomely
    </h1>
  </header>
  <p
    className="intro"
  >
    To get started, edit 
    <code>
      src/client/components/App.js
    </code>
     and save to reload.
  </p>
  <p
    className="styles"
  >
    You can edit any styling in 
    <code>
      src/client/sass/index.scss
    </code>
    .
  </p>
</div>
`;
